packages/layout/src/components/helplink/helplink.tsx

```diff
@@ -2,28 +2,55 @@ import React from 'react';
 import Collapsible from 'react-collapsible';
 import { Hint } from '../hint/hint';
 import { ArrowDown } from '@tpr/icons';
-import { Flex } from '@tpr/core';
 import Styles from './helplink.module.scss';
 
 type HelpLinkProps = {
 	title: string;
-	content: string;
+	className?: string;
+	triggerClassName?: string;
+	contentClassName?: string;
 };
 
-export const HelpLink: React.FC<HelpLinkProps> = ({ title, content }) => {
+export const HelpLink: React.FC<HelpLinkProps> = (props) => {
+  // Use a separate component for the trigger so that we can set aria-expanded, and place the inline SVG inside the button
+  const [expanded, setExpanded] = React.useState(false);
+  const Trigger = () => (
+    <button aria-expanded={expanded}>
+      <ArrowDown />
+			{props.title}
+    </button>
+  );
+
+  // Create an alert when the component is expanded/collapsed, because the trigger button loses focus which prevents the change to aria-expanded being announced.
+  // Using a ref to get the button element and call its focus() method doesn't work.
+  const [accessibleAlert, setAccessibleAlert] = React.useState<JSX.Element>();
+  function createAccessibleAlert() {
+    return (
+      <p role="alert" className={Styles.visuallyHidden}>
+        {expanded ? 'collapsed' : 'expanded'}
+      </p>
+    );
+	}
+	
 	return (
+    <>
 		<Collapsible
-			trigger={
-				<Flex>
-					<ArrowDown />
-					{title}
-				</Flex>
-			}
-			transitionTime={400}
-			triggerClassName={Styles.closed}
-			triggerOpenedClassName={Styles.open}
+			trigger={<Trigger />}
+			classParentString={`${Styles.helpLink} ${props.className}`.trim()}
+			contentInnerClassName={`${Styles.helpLinkContent} ${props.contentClassName}`.trim()}
+			overflowWhenOpen="visible"
+			onOpening={() => {
+				setExpanded(true);
+				setAccessibleAlert(createAccessibleAlert());
+			}}
+			onClosing={() => {
+				setExpanded(false);
+				setAccessibleAlert(createAccessibleAlert());
+			}}
 		>
-			<Hint>{content}</Hint>
+			<Hint>{props.children}</Hint>
```

Comment:
with my other comment, I mean something like:
```
<Hint>
  <p>{props.content}</p>
</Hint>
```

Reply:
I don't think so. The reason I moved it was because it didn't support HTML, eg a bulleted list on an example I'm working with. I can look for existing examples and ensure they're updated.

Reply:
Ok, in that case you only need to update the tests for the circleci checks to pass.  Currently is receiving `content` in the props.